### PR TITLE
Doc 4135

### PR DESCRIPTION
**Pull request by colin-hillis**
- All headings in DP2 now using title case
- Decentralised > Decentralized (US English)
- Second heading on first-flow.md moved from H1 to H2 so that it displays

### CODE REVIEW

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 date: '2022-09-20'
-title: "Fast feedback with Simulator"
+title: "Fast Feedback With Simulator"
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md:3]:
```suggestion
title: "Fast Feedback with Simulator"
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md (hunk 1 above)
Comment (colin-hillis) [reply content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md:3]: The common approach is that 'with' is capitalised, as it is 4 letters+ and doesn't fit into the other exclusion categories.

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/first-flow.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ In the responder flow:
     @Suspendable
     override fun call(session: FlowSession) { ... }
 ```
-## Injecting service
+## Injecting Service
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-dev-preview-2/getting-started/first-flow.md:245]:
```suggestion
## Injecting Services
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
Thanks @colin-hillis

**Pull request merged**